manifests/repo.pp

```diff
@@ -1,9 +1,23 @@
 # private class
 class sensu::repo {
 
-  if $::sensu::repo_class {
-    contain $::sensu::repo_class
+  case $::facts['osfamily'] {
+    'RedHat': {
+      # TODO: change from nightly to stable once there are stable releases
+      yumrepo { 'sensu_nightly':
+        baseurl         => "https://packagecloud.io/sensu/nightly/el/${::operatingsystemmajrelease}/\$basearch",
+        repo_gpgcheck   => 1,
+        gpgcheck        => 0,
+        enabled         => 1,
+        gpgkey          => 'https://packagecloud.io/sensu/nightly/gpgkey',
+        sslverify       => 1,
+        sslcacert       => '/etc/pki/tls/certs/ca-bundle.crt',
+        metadata_expire => 300,
```

Comment:
before => Package

Reply:
The reason I didn't do that is because the packages have `require => Class['sensu::repo']` since using before would only catch something like `sensu-agent` and would not work for `sensu-backend` and `sensu-cli`.

Reply:
aha.. let's continue with requiring the class and contains then